webpack.common.js

```diff
@@ -62,6 +62,8 @@ module.exports = {
       PXT_HEX_URL: JSON.stringify(process.env.PXT_HEX_URL
         || 'https://rovercode-pxt.s3.us-east-2.amazonaws.com/alpha/rovercode.hex'),
       SAVE_DEBOUNCE_TIME: JSON.stringify(process.env.SAVE_DEBOUNCE_TIME),
+      SUBSCRIPTION_SERVICE: JSON.stringify(process.env.SUBSCRIPTION_SERVICE
```

Comment:
I was going to suggest that we define this per branch in the .travis.yml deploys, but I guess that's [not a thing](https://stackoverflow.com/questions/43210945/travis-ci-how-to-set-environment-variables-for-deployment-script-only). We can just set them in the Travis build settings in their web interface.